plugins/cisco_umbrella_enforcement/komand_cisco_umbrella_enforcement/actions/add_event/schema.py

```diff
@@ -8,178 +8,175 @@ class Component:
 
 
 class Input:
-    EVENTS = "events"
+    ALERTTIME = "alertTime"
+    DEVICEID = "deviceId"
+    DEVICEVERSION = "deviceVersion"
+    DISABLEDSTSAFEGUARDS = "disableDstSafeguards"
+    DSTDOMAIN = "dstDomain"
+    DSTIP = "dstIP"
+    DSTURL = "dstUrl"
+    EVENTDESCRIPTION = "eventDescription"
+    EVENTHASH = "eventHash"
+    EVENTSEVERITY = "eventSeverity"
+    EVENTTIME = "eventTime"
+    EVENTTYPE = "eventType"
+    EXTERNALURL = "externalURL"
+    FILEHASH = "fileHash"
+    FILENAME = "fileName"
+    PROTOCOLVERSION = "protocolVersion"
+    PROVIDERNAME = "providerName"
+    SRC = "src"
     
 
 class Output:
     ID = "ID"
     
 
-class AddEventInput(komand.Input):
+class AddEventInput(insightconnect_plugin_runtime.Input):
     schema = json.loads("""
    {
   "type": "object",
   "title": "Variables",
   "properties": {
-    "events": {
-      "type": "array",
-      "title": "Events",
-      "description": "Generic event format field. More info at https://docs.umbrella.com/developer/enforcement-api/generic-event-format2/",
-      "items": {
-        "$ref": "#/definitions/event"
-      },
+    "alertTime": {
+      "type": "string",
+      "title": "Alert Time",
+      "description": "The time the event was sent to Umbrella",
       "order": 1
+    },
+    "deviceId": {
+      "type": "string",
+      "title": "Device ID",
+      "description": "The ID of the device sending the event",
+      "order": 2
+    },
+    "deviceVersion": {
+      "type": "string",
+      "title": "Device Version",
+      "description": "The version of the device sending the event",
+      "order": 3
+    },
+    "disableDstSafeguards": {
+      "type": "boolean",
+      "title": "Disable Destination Safeguards",
+      "description": "True bypasses validations normally performed against submitted events",
+      "default": false,
+      "order": 18
+    },
+    "dstDomain": {
+      "type": "string",
+      "title": "Destination Domain",
+      "description": "The destination domain specified following RFC 3986 encoding guidelines",
+      "order": 4
+    },
+    "dstIP": {
+      "type": "string",
+      "title": "Destination IP",
+      "description": "The destination UP of the domain, specified in IPv4 dotted-decimal notation",
+      "order": 5
+    },
+    "dstUrl": {
+      "type": "string",
+      "title": "Destination URL",
+      "description": "The destination URL specified following RFC 3986 encoding guidelines",
```

Comment:
Suggested reword: The destination domain specified (follow RFC 3986 encoding guidelines)